SPECS/sympa-6.2.spec.in

```diff
@@ -529,6 +534,20 @@ getent passwd sympa >/dev/null || \
       -c "System User for Sympa" \
       -s "/sbin/nologin" \
       sympa
+# Fix CSS and pictures paths
+if [ $1 -gt 1 ]; then
+    if [ -d %{_localstatedir}/lib/%{name}/static_content/css -o \
+         -d %{_localstatedir}/lib/%{name}/static_content/pictures ]; then
+        cp -pur %{_localstatedir}/lib/%{name}/static_content/{css,pictures} \
+            %{_localstatedir}/lib/%{name}/
+        echo
+        echo The css and pictures directories were moved to
+        echo %{_localstatedir}/lib/%{name}/
+        echo Please delete old ones from
+        echo %{_localstatedir}/lib/%{name}/static_content/
+        echo
+    fi
+fi
```

Comment:
If the user updated to this version, not had removed older css & pictures directories, and updated to the next version, older directories will be copied again and pictures may be overwritten.

How about moving older css & pictures directories?

Reply:
I missed `-u` of `cp`, and problem is slighter than I wrote at first: Even if user (administrator) removed pictures from new directory, they may be restored from older directory.

Reply:
I thought about using ```mv``` rather than ```cp```, but ```cp``` seemed safer in case anything would go wrong with the copy. Indeed, the case you mentioned will not be handled properly in case the admin doesn't actually remove the old data and keep working with them. Another issue I see with using ```cp``` is it would take twice as much space on the filesystem and that could be an issue, while ```mv``` won't have this problem.
I have a mixed feeling, there don't seem to be 100% safe solution. Anyway, ```mv``` seems to be better than ```cp```. I'll change the code to do that and I'll remove the message which isn't useful anymore with moving the data rather than copying it.